benchmark-scripts/requirements-xpu.txt

```diff
@@ -1,8 +1,8 @@
 Flask==3.0.2
 Flask-HTTPAuth==4.8.0
 requests==2.31.0
-prometheus-client==0.19.0
-grpcio==1.60.1
-protobuf==4.25.2
-marshmallow==3.20.2
+prometheus-client==0.20.0
+grpcio==1.62.1
+protobuf==5.26.0
```

Comment:
is unit test still working for this new protobuf update?

Reply:
It was on my local box but I can validate on the integration system

Reply:
Confirmed the new protobuf causes no issues